### DISCUSSION TITLE
AlwaysOn - Exclude databases?

### DISCUSSION THREAD

**Discussion by HansLindgren** [trimble-oss/dba-dash#718] [General]
Hello!

I would like to exclude 1+ databases that are in my availability groups... Is there a way of excluding the check for their 'replication health'?
I.e. I get this:
![image](https://github.com/trimble-oss/dba-dash/assets/6190303/421dc483-052b-458f-95ed-a3037c862598)
and I prefer not having error status for these availability groups.
(The long reason, I have this issue, is that One database (on multiple instances), that are part of Distributed Availability Groups, should not be replicated in my environments)

Regards,
Hans

**Comment by DavidWiseman** [comment]
Hi.  There isn't a way to exclude it at the moment.  It seems like a bit of an unusual setup so I'm not sure how many people would benefit from this, but I'll keep it in mind for future development.
